run-test.sh

```diff
@@ -253,8 +250,6 @@ coreclr_code_coverage()
 # Parse arguments
 
 ((serverGC = 0))
-OuterLoop="-notrait category=outerloop"
-IgnoreForCI=""
```

Comment:
Where do RunTests.sh/RunTests.cmd come from? It looks like they're generated by some build targets (https://github.com/dotnet/buildtools/pull/649/ ?) but it isn't clear to me what exactly their purpose is.

Also, does this remove our ability to pass Outerloop and IgnoreForCI explicitly from the netci.groovy for running tests? Because they're removed from run-test.sh here but they're still being passed in as input in the netci file (line 448).